### PR TITLE
assignment submit

### PR DESCRIPTION
**Pull request by LeviZ**

### CODE REVIEW

**Comment by LeviZ** [comment]
added responsive-and-assets assignment on last commit

File: LeviZortman-Muhammed-Safa-Yasar/advanced-css-layout/css/base.css [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  font-size: 3.40em;
+  background: -webkit-linear-gradient(#eee, #333);
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+}
+
+p {
+  font-size: 1em;
+}
+
```
Comment (litteredwitherrors) [inline LeviZortman-Muhammed-Safa-Yasar/advanced-css-layout/css/base.css:36]: Remove all the extra spaces below.

File: LeviZortman-Muhammed-Safa-Yasar/advanced-css-layout/css/layout.css [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+  content:"";
+  display:table;
+}
+.group:after {
+  clear:both;
+}
+.group {
+    zoom:1; /* For IE 6/7 */
+}
+
```
Comment (litteredwitherrors) [inline LeviZortman-Muhammed-Safa-Yasar/advanced-css-layout/css/layout.css:44]: Some rules have no space between them:

```
selector {
  property: value;
}
selector {
  property: value;
}
```

And some do:

```
selector {
  property: value;
}

selector {
  property: value;
}
```

Either way is fine but keep it consistent.

File: LeviZortman-Muhammed-Safa-Yasar/advanced-css-layout/css/modules.css [hunk 3]
```diff
@@ -0,0 +1,65 @@
+
```
Comment (litteredwitherrors) [inline LeviZortman-Muhammed-Safa-Yasar/advanced-css-layout/css/modules.css:1]: Clean up extra empty lines in this file.

File: LeviZortman-Muhammed-Safa-Yasar/advanced-css-layout/index.html [hunk 4]
```diff
@@ -0,0 +1,163 @@
+<DOCTYPE html>
+<html>
+  <head>
+  <link rel="stylesheet" href="css/normalize.min.css">
+  <link rel="stylesheet" href="css/base.css">
+  <link rel="stylesheet" href="css/layout.css">
+  <link rel="stylesheet" href="css/modules.css">
+  <link rel="stylesheet" href="css/fonts.css">
+
```
Comment (litteredwitherrors) [inline LeviZortman-Muhammed-Safa-Yasar/advanced-css-layout/index.html:9]: Remove extra line.

File: LeviZortman-Muhammed-Safa-Yasar/project-to-refactor/style.css [hunk 5]
```diff
@@ -0,0 +1,112 @@
+
```
Comment (litteredwitherrors) [inline LeviZortman-Muhammed-Safa-Yasar/project-to-refactor/style.css]: What is this file for?

File: LeviZortman-Muhammed-Safa-Yasar/project-to-refactor/index.html [hunk 6]
```diff
@@ -0,0 +1,149 @@
+<DOCTYPE html>
```
Comment (litteredwitherrors) [inline LeviZortman-Muhammed-Safa-Yasar/project-to-refactor/index.html]: Duplicate index.html?

**Comment by litteredwitherrors** [comment]
It appears that you have duplicate index.html and style.css files. Remove the unnecessary files.

**Pull request closed without merging**